docs/synapse/userguides/storm_adv_control.rstorm

```diff
@@ -115,27 +166,112 @@ You can print the value of a variable, to check its value at a given point in yo
   $asn=:asn
   $lib.print($asn)
 
-You can also print values associated with the node(s) in the current working set, using the various methods associated with the ``$node`` Storm type. (See :ref:`storm-adv-methods` for a user-focused introduction to methods, or :ref:`stormprims-storm-node-f527` in the detailed Storm Libraries / Storm Types documentation for a more technical discussion.)
+You can also print values associated with the node(s) in the current working set, using the various methods
+associated with the ``$node`` Storm type. (See :ref:`storm-adv-methods` for a user-focused introduction to methods,
+or :ref:`stormprims-storm-node-f527` in the detailed Storm Libraries / Storm Types documentation for a more
+technical discussion.)
 
 ::
   
   $lib.print($node.ndef())
 
 
-See the :ref:`storm-adv-example` below for an illustration of some of these effects.
-
+.. _storm-adv-control-flow:
 
 Control Flow Operations
 -----------------------
 
+.. TIP::
+  
+  The examples below are Storm excerpts used to illustrate specific concepts, but do not represent
+  complete Storm queries / Storm code.
+
+.. _flow-init:
+
+Init Block
+++++++++++
+
+An **init block** allows you to execute the specified Storm **once** at the beginning of your overall
+Storm query. This allows you to use Storm to perform a set of initial operations "outside" of the normal
```

Comment:
outside is kind of a weird word to use.
init blocks are technically run prior to any nodes having the opportunity to pass through the code containing the block; so they are usually used to setup variables / import libraries / etc.